rst_files/wald_friedman.rst

```diff
@@ -185,57 +168,24 @@ Activate the project environment, ensuring that ``Project.toml`` and ``Manifest.
 
 .. code-block:: julia
 
-  using Distributions, Plots, LaTeXStrings, LinearAlgebra, QuantEcon, Printf, Interpolations
```

Comment:
Did the example source here get deleted, or is the small code block below sufficient to replicate example?